src/main/kotlin/io/polygon/kotlin/sdk/websocket/PolygonWebSocketSubscription.kt

```diff
@@ -24,21 +24,52 @@ sealed class PolygonWebSocketChannel(val prefix: String) {
         object AggPerSecond : Stocks("A")
         object AggPerMinute : Stocks("AM")
         object AggPerMinuteLaunchpad : Stocks("AM")
-        object LaunchpadValue : Launchpad("LV")
+    }
+
+    /**
+     * Usable in the [PolygonWebSocketCluster.LaunchpadStocks] cluster
+     */
+    sealed class LaunchpadStocks(channelPrefix: String) : PolygonWebSocketChannel(channelPrefix) {
+        object AggPerMinute : LaunchpadStocks("AM")
+        object LaunchpadValue : LaunchpadStocks("LV")
+    }
+
+    /**
+     * Usable in the [PolygonWebSocketCluster.Options] cluster
+     */
+    sealed class Options(channelPrefix: String) : PolygonWebSocketChannel(channelPrefix) {
+        object Trades: Options("T")
+        object Quotes: Options("Q")
+        object AggPerSecond : Options("A")
+        object AggPerMinute : Options("AM")
+    }
+
+    /**
+     * Usable in the [PolygonWebSocketCluster.LaunchpadOptions] cluster
+     */
+    sealed class LaunchpadOptions(channelPrefix: String) : PolygonWebSocketChannel(channelPrefix) {
+        object AggPerMinute : LaunchpadOptions("AM")
+        object LaunchpadValue : LaunchpadOptions("LV")
     }
 
     /**
      * Usable in the [PolygonWebSocketCluster.Forex] cluster
      */
     sealed class Forex(channelPrefix: String) : PolygonWebSocketChannel(channelPrefix) {
         object Quotes: Forex("C")
-        object AggPerSecond: Forex("CAS")
+        object AggPerSecond : Forex("CAS")
         object AggPerMinute: Forex("CA")
         object Bonds: Forex("BONDS")
         object Commodities: Forex("COMMODITIES")
         object Metals: Forex("METALS")
```

Comment:
We should remove these 3 while we're making a breaking change, they haven't been supported for a while
```suggestion
```